packages/components/tests/dummy/app/templates/components/link-to/standalone.hbs

```diff
@@ -175,74 +121,154 @@
         <li>trailing</li>
       </ol>
     </dd>
-    <dt>color <code>enum</code></dt>
+    <dt>route <code>route name</code></dt>
+    <dd>The route that is passed to the link</dd>
+    <dt>model/models <code>route model(s)</code></dt>
+    <dd>Use this to explicitly supply a model to the route</dd>
+    <dt>query <code>query params as key value pairs</code></dt>
+    <dd>Use this to supply query parameters to the route</dd>
+    <dt>“splattributes”</dt>
     <dd>
-      <p>Acceptable values:</p>
-      <ol>
-        <li class="default">primary</li>
-        <li>secondary</li>
-      </ol>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
     </dd>
   </dl>
 </section>
 
 <section>
-  <h3 class="dummy-h3">
-    Usage
-  </h3>
+  <h3 class="dummy-h3">Usage</h3>
 
-  <h4 class="dummy-h4">Basic invocation</h4>
+  <h4 class="dummy-h4">Basic use</h4>
+  <p class="dummy-paragraph">The most basic invocation requires both
+    <code class="dummy-code">icon</code>
+    and
+    <code class="dummy-code">text</code>
+    to be passed and the routes details (for details on how routing is handled see below).</p>
   {{! prettier-ignore-start }}
   <CodeBlock
     @language="markup"
     @code='
-      <Hds::LinkTo::Standalone @text="Watch tutorial video" @icon="film" @route="index" />
+      <Hds::LinkTo::Standalone @icon="film" @text="Watch tutorial video" @route="..." />
     '
   />
   {{! prettier-ignore-end }}
   <p class="dummy-paragraph">Renders to:</p>
-  <Hds::LinkTo::Standalone @text="Watch Tutorial Video" @icon="film" @route="index" />
+  <Hds::LinkTo::Standalone @icon="film" @text="Watch tutorial video" @route="components.link-to.standalone" />
 
-  <h4 class="dummy-h4">Sizes</h4>
-  <p class="dummy-paragraph">There are three sizes: small, medium, and large. If no size is defined,
-    <code class="dummy-code">medium</code>
-    is automatically applied.
+  <h4 class="dummy-h4">Icon position</h4>
+  <p class="dummy-paragraph">By default the icon is placed before the text. If you would like to position it after the
+    text, define
+    <code class="dummy-code">@iconPosition</code>:
   </p>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::LinkTo::Standalone @icon="film" @iconPosition="trailing" @text="Watch tutorial video" @route="..." />
+    '
+  />
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::LinkTo::Standalone
+    @icon="film"
+    @iconPosition="trailing"
+    @text="Watch tutorial video"
+    @route="components.link-to.standalone"
+  />
 
-  <h5 class="dummy-h5">Small</h5>
+  <h4 class="dummy-h4">Color</h4>
```

Comment:
[question, maybe issue] does this belong here? It's referring to the button component.

Reply:
For some reason I can't add a suggestion below, but I think what Melanie means is to change L180 to "for a LinkTo (Standalone) instead of "for a button"

Reply:
Ok, I see. No problem. Fixed.